### ISSUE TITLE
Changed dkp kill value but didn’t work

### ISSUE THREAD

**Issue by Mixxcorpserun** [Roeshambo/MonolithDKP#327]
Tried to change the dkp value/kill to 5/kill instead of 10 do MC and it didn’t work. It kept awarding 10. Any idea what I’m doing wrong?

**Comment by Roeshambo** [comment]
Did you save the settings at the bottom after changing?

**Comment by Mixxcorpserun** [comment]
Yup I figured it out. But I had another question. Where is the best place to download the add on? Some people in my guild can’t get it working at all.

**Comment by Roeshambo** [comment]
Best place would be curse. If it's not working for them, have them close the game and delete the saved variables file and an officer can send them fresh data.
